src/utils/health_check.py

```
@@ -0,0 +1,77 @@
+"""Health check utilities for service dependencies."""
+
+import asyncio
+import logging
+import aiohttp
+from typing import Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)